### DISCUSSION TITLE
Should addons include Scratch in the name?

### DISCUSSION THREAD

**Discussion by jeffalo** [ScratchAddons/ScratchAddons#95] [Discussion]
hello, so i noticed that a few addons (including mine) include "scratch" in the name.

![image](https://user-images.githubusercontent.com/40470736/90973550-5459aa80-e523-11ea-9a8e-206cf804ca8f.png)

is this really necessary? since scratchaddons would be targeted for scratch, the user would expect addons to be scratch related so it kinda is a waste of space/time spent reading to have scratch there?

**Comment by WorldLanguages** [comment]
Yeah I see.
Scratch Notifier and Scratch 3 developer tools probably deserve "Scratch" in their names because they are originally somewhat popular extensions with "Scratch" in the name (and now I see, "Developer Tools" should be in caps).
View stepping and image uploader don't make much sense. They should probably be renamed to something like "Editor view stepping" or "block view stepping" and "Forums image uploader"

**Comment by Hans5958** [comment]
I don't think you need to have "Scratch" in the name, since in the end, I think the code would be different than the original source code as time goes on. Who knows? Someday there will be a rewrite that makes the code really different than the original. Also, it's for uniformity purposes.

But again, it's up to the original developers, and probably licenses, I guess.
